hist/histv7/inc/ROOT/RHistData.hxx

```diff
@@ -369,6 +434,46 @@ public:
       for (size_t b = 0; b < fOverflowSumWeightsSquared.size(); ++b)
          fOverflowSumWeightsSquared[b] += other.fOverflowSumWeightsSquared[b];
    }
+
+   /// Multiply by a scalar.
+   template <typename T>
+   void MultiplyByScalar(const T scalar) {
+      for (size_t b = 0; b < fSumWeightsSquared.size(); ++b)
+         fSumWeightsSquared[b] *= scalar;
+      for (size_t b = 0; b < fOverflowSumWeightsSquared.size(); ++b)
+         fOverflowSumWeightsSquared[b] *= scalar;
+   }
+
+   /// Divide by an other `RHistStatUncertainty` data, assuming same bin configuration.
+   void Divide(const RHistStatUncertainty& other) {
+      assert(fSumWeightsSquared.size() == other.fSumWeightsSquared.size());
+      assert(fOverflowSumWeightsSquared.size() == other.fOverflowSumWeightsSquared.size());
+      for (size_t b = 0; b < fSumWeightsSquared.size(); ++b)
+         fSumWeightsSquared[b] /= other.fSumWeightsSquared[b];
+      for (size_t b = 0; b < fOverflowSumWeightsSquared.size(); ++b)
+         fOverflowSumWeightsSquared[b] /= other.fOverflowSumWeightsSquared[b];
+      // for (size_t b = 0; b < fSumWeightsSquared.size(); ++b)
```

Comment:
Is this part of the discussion "divide bin content before or after dividing other stats"?